src/main/java/edu/cornell/kfs/coa/businessobject/CuAccountGlobal.java

```diff
@@ -2,164 +2,869 @@
+		}
+		
+		if(extrnlFinEncumSufficntFndIndicator != null){
+			account.setExtrnlFinEncumSufficntFndIndicator(extrnlFinEncumSufficntFndIndicator);
+		}
+		
+		if(intrnlFinEncumSufficntFndIndicator != null){
+			account.setIntrnlFinEncumSufficntFndIndicator(intrnlFinEncumSufficntFndIndicator);
+		}
+		
+		if(finPreencumSufficientFundIndicator != null){
+			account.setFinPreencumSufficientFundIndicator(finPreencumSufficientFundIndicator); 
+		}
+		
+		if(financialObjectivePrsctrlIndicator != null){
+			account.setFinancialObjectivePrsctrlIndicator(financialObjectivePrsctrlIndicator);
+		}
+
+		if (StringUtils.isNotBlank(contractControlFinCoaCode)) {
+		    account.setContractControlFinCoaCode(contractControlFinCoaCode);
+		}
+		
+		if (StringUtils.isNotBlank(contractControlAccountNumber)) {
+		    account.setContractControlAccountNumber(contractControlAccountNumber);
+		}
+		
+		if (StringUtils.isNotBlank(acctIndirectCostRcvyTypeCd)) {
+		    account.setAcctIndirectCostRcvyTypeCd(acctIndirectCostRcvyTypeCd);
+		}
+
+		if (StringUtils.isNotBlank(financialIcrSeriesIdentifier)) {
+		    account.setFinancialIcrSeriesIdentifier(financialIcrSeriesIdentifier);
+		}
+		
+		if (ObjectUtils.isNotNull(contractsAndGrantsAccountResponsibilityId)) {
+		    account.setContractsAndGrantsAccountResponsibilityId(contractsAndGrantsAccountResponsibilityId);
+		}
+
+		if (StringUtils.isNotBlank(invoiceFrequencyCode)) {
+		    ((AccountExtendedAttribute) account.getExtension()).setInvoiceFrequencyCode(invoiceFrequencyCode);
+		}
+		
+		if (StringUtils.isNotBlank(invoiceTypeCode)) {
+		    ((AccountExtendedAttribute) account.getExtension()).setInvoiceTypeCode(invoiceTypeCode);
+		}
+
+		if (everify != null) {
+		    ((AccountExtendedAttribute) account.getExtension()).setEverify(everify);
+		}
+		
+		if (ObjectUtils.isNotNull(costShareForProjectNumber)) {
+		    ((AccountExtendedAttribute) account.getExtension()).setCostShareForProjectNumber(costShareForProjectNumber);
+		}
+		
+		if(removeAccountExpirationDate){
+			account.setAccountExpirationDate(null);
+		}
+		
+		if(removeContinuationChartAndAccount){
+			account.setContinuationFinChrtOfAcctCd(null);
+			account.setContinuationAccountNumber(null);
+		}
+		
+		if(removeIncomeStreamChartAndAccount){
+			account.setIncomeStreamFinancialCoaCode(null);
+			account.setIncomeStreamAccountNumber(null);
+		}
+	}
+
+	private void updateAccountExtendedAttribute(Account account) {
+		AccountExtendedAttribute aea = (AccountExtendedAttribute) (account.getExtension());
+		if (this.getClosed() != null && this.getClosed() && aea.getAccountClosedDate() == null) {
+		    aea.setAccountClosedDate(SpringContext.getBean(DateTimeService.class).getCurrentSqlDate());
+		} else if (this.getClosed() != null && !this.getClosed() && aea.getAccountClosedDate() != null) {
+		    aea.setAccountClosedDate(null);           
+		}
+	}
+	
+    public List<IndirectCostRecoveryAccountChange> getActiveIndirectCostRecoveryAccounts() {
```

Comment:
The indentation seems off in many of these files. Maybe there is a tabs vs. spaces issue, but it would be nice if it was cleaned up and made consistent in all these files.